spec/requests/legacy/redirect_spec.rb

```diff
@@ -46,4 +46,13 @@
       expect(response.status).to eql 301
     end
   end
+
+  describe 'cookies page' do
+    it 'redirects to the cookies page' do
+      get '/cookies-policy'
+
+      expect(response).to redirect_to(cookies_url)
+      expect(response.status).to eql 301
```

Comment:
The [`have_http_status`](https://relishapp.com/rspec/rspec-rails/docs/matchers/have-http-status-matcher) matcher is a more expressive way of specifying this.

```rb
expect(response).to have_http_status(:moved_permanently)
```

See also #392